fwupdate/source/cc32xx/platform/cc32xx/cc32xx_support.c

```diff
@@ -0,0 +1,758 @@
+/*
+ * Copyright 2014-2015 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <time.h>
+#include "cc32xx_support.h"
+
+#define DEMO_UNUSED(x) if(x){}
+
+#define HTTP_BUF_LEN 1440
+#define FIRMWARE_FILE_PATH "/sys/mcuimg1.bin"
```

Comment:
If this filename does not match to default, please document this in installation instructions.